server.js

```diff
@@ -213,4 +220,251 @@ app.get('/consensus/voted_power', function (req, res) {
+          if (data1.data.them)
+          {
+            console.log(data1.data.them[0].pictures.primary.url)
+          }
+          let avatar;
+
+          if (data1.data.them)
+          {
+            avatar = data1.data.them[0].pictures.primary.url
+          }
+          else {
+            avatar = 'none'
+          }
+          output[0].push({ image: avatar });
+
+          if (data2.data.them)
+          {
+            avatar = data2.data.them[0].pictures.primary.url
+          }
+          else {
+            avatar = 'none'
+          }
+          output[1].push({ image: avatar });
+
+          if (data3.data.them)
+          {
+            avatar = data3.data.them[0].pictures.primary.url
+          }
+          else {
+            avatar = 'none'
+          }
+          output[2].push({ image: avatar });
+
+          if (data4.data.them)
+          {
+            avatar = data4.data.them[0].pictures.primary.url
+          }
+          else {
+            avatar = 'none'
+          }
+          output[3].push({ image: avatar });
+
+          if (data5.data.them)
+          {
+            avatar = data5.data.them[0].pictures.primary.url
+          }
+          else {
+            avatar = 'none'
+          }
+          output[4].push({ image: avatar });
+
+          if (data6.data.them)
+          {
+            avatar = data6.data.them[0].pictures.primary.url
+          }
+          else {
+            avatar = 'none'
+          }
+          output[5].push({ image: avatar });
+
+          if (data7.data.them)
+          {
+            avatar = data7.data.them[0].pictures.primary.url
+          }
+          else {
+            avatar = 'none'
+          }
+          output[6].push({ image: avatar });
+
+          if (data8.data.them)
+          {
+            avatar = data8.data.them[0].pictures.primary.url
+          }
+          else {
+            avatar = 'none'
+          }
+          output[7].push({ image: avatar });
+
+          if (data9.data.them)
+          {
+            avatar = data9.data.them[0].pictures.primary.url
+          }
+          else {
+            avatar = 'none'
+          }
+          output[8].push({ image: avatar });
+
+          if (data10.data.them)
+          {
+            avatar = data10.data.them[0].pictures.primary.url
+          }
+          else {
+            avatar = 'none'
+          }
+          output[9].push({ image: avatar });
+
+          var output_str = ''
+          for (let i = 0; i < output.length; i++)
+          {
+            output_str += output[i][0].name + ',' + output[i][1].image + ',' + output[i][0].power + ',' + output[i][0].uptime
+            if (i < 9)
+            {
+              output_str += ','
+            }
+          }
+          res.send(output_str);
+        }));
+      }));
+    });
+});
+
 app.listen(PORT, () => console.log(`Listening on port ${PORT}!`));
+
+
+function addToOuptut(message)

```

Comment:
What is this function for? It doesn't seem to be used and is spelled incorrectly I believe?

Reply:
it's remnant from old variation of the code. I just forgot to delete it.